pkg/securitypolicy/securitypolicy_test.go

```diff
@@ -38,9 +39,16 @@ const (
 var testRand *rand.Rand
 
 func init() {
-	seed := rand.NewSource(time.Now().Unix())
-	testRand = rand.New(seed)
-	fmt.Fprintf(os.Stdout, "securitypolicy_test seed: %d\n", seed.Int63())
+	seed := time.Now().Unix()
+	if seedStr, ok := os.LookupEnv("SEED"); ok {
+		var err error
+		seed, err = strconv.ParseInt(seedStr, 10, 64)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to parse seed: %d\n", seed)
+		}
```

Comment:
If we fail to parse you're going to assign seed to 0 and then use it as the source below. We should only reassign seed if this succeeds